### ISSUE TITLE
Is it normal to have a testing accuracy 0.001 on ImageNet after two days of training (iteration: 110, 000)?

### ISSUE THREAD

**Issue by wangdelp** [BVLC/caffe#2085]
I am training ImageNet model using the default configuration provided in CAFFE. But sounds like the objective function did not go down much, and the testing accuracy did not improve after two days of training on K40 (current iteration is around 110, 000). Suppose in the first few iterations the model should improve quite fast. But what I have done is just follow the default configuration without any modification. Is that normal? Thanks.

**Comment by lugiavn** [comment]
No, 0.001 is random guess performance, your network isn't learning anything.
Open the network file and change all the bias 1 to 0.1 or something, it helped when I did it. You should see training loss decreasing after 2000 iterations or so.

**Comment by jltmtz** [comment]
See #59
Also, you might just want to restart a few times and one trial will eventually start learning. As a rule of thumb, the loss should decrease after 15,000 iterations.
The question seems to keep popping up. It might be a good idea to include a note in the tutorial page...
